Tests/docker_native_image_config.json

```diff
@@ -30,6 +30,36 @@
             "tesseract"
          ],
          "docker_ref": "demisto/py3-native:8.1.0.40949"
+      },
+      "native:dev":{
```

Comment:
before you merge make sure that if there isn't `docker_ref` to a native image (like in the native:dev) that it will not break the code in the SDK.

Reply:
It works :) thanks.